.mergify.yml

```
@@ -49,10 +49,10 @@ queue_rules:
               - "status-success=ci/centos/jjb-validate"
 
 pull_request_rules:
-  - name: start CI jobs for queued PR
+  - name: start CI jobs for merge queue PRs by Mergify
     conditions:
       - base~=^(devel)|(release-.+)$
-      - "check-pending=Queue: Embarked in merge train"
+      - author=mergify
```

```suggestion
      - or:
        - author=mergify
        - "check-pending=Queue: Embarked in merge train"
```

I think we need one of either checks to pass for adding ok-to-test?

The second check is still required for upto date pr with only one pr in queue / the original pr even if mergifyio does speculative checks ?

---

The 2nd check would not need to add the `ok-to-test` label if the PR is up to date. The `Queue: Embarked in merge train` message only appears when all required CI jobs have passed.